### DISCUSSION TITLE
Expression Issue

### DISCUSSION THREAD

**Discussion by armenasadyan** [microsoft/RulesEngine#363] [General]
Hi,
For some reasons I need to use version 3.1.0.0.

  {
    "WorkflowName": "inputWorkflow",
    "Rules": [
      {
        "RuleName": "GiveDiscount10Percent",
        "SuccessEvent": "10",
        "ErrorMessage": "One or more adjust rules failed.",
        "ErrorType": "Error",
        "RuleExpressionType": "LambdaExpression",
        "Expression": "input1.country == \"usa\""
      }
    ]
  }

When I execute rule, I have exception message: Exception while parsing expression `input1.country == "usa"` - Method not found: 'System.Func`2<!!0,!!1> FastExpressionCompiler.ExpressionCompiler.CompileFast(System.Linq.Expressions.Expression`1<System.Func`2<!!0,!!1>>, Boolean)'.

Please advise.

Thank you.

**Comment by abbasc52** [comment]
It looks like dependency issue. It says it is unable to find `FastExpressionCompiler.ExpressionCompiler.CompileFast` method (at runtime). You might want to check if you have a different version of FastExpressionCompiler in your code and you might be able to fix it by adding the correct version of FastExpressionCompiler (>= 2.0.0) in your dependency.

Let me know if this helps.

**Reply by armenasadyan** [reply]
Thank you Abbas, that works.
